app/util/cookies.ts

```diff
@@ -14,7 +14,7 @@ import HarmonyRequest from '../models/harmony-request';
 import { get } from 'lodash';
 import { randomBytes } from 'crypto';
 
-export const cookieOptions = { signed: true, sameSite: 'Lax' };
+export const cookieOptions = { signed: true, sameSite: 'lax' as const };
```

Comment:
What does this change do?

Reply:
It forces 'lax' to be a specific type instead of just a string to match the declaration in the node module. TypeScript was complaining about it.